mne/simulation/source.py

```diff
@@ -489,16 +510,17 @@ def get_stc(self, start_sample=0, stop_sample=None):
             wf_stop = self._last_samples[ind[i]]
 
             # Recover the indices of the event that should be in the chunk
-            waveform_ind = np.in1d(np.arange(wf_start, wf_stop),
-                                   np.arange(start_sample, stop_sample))
+            waveform_ind = np.in1d(np.arange(wf_start, wf_stop + 1),
+                                   np.arange(start_sample, stop_sample + 1))
```

Comment:
why these +1? with zero-based indexing it should not happen

Reply:
The stop_sample is interpreted here as the (0-based) index of the last sample, analogous to the .last_samp property of raw objects.  So the +1 is necessary to make sure the chosen samples span the start to stop sample, inclusive.  e.g. np.arange(0, 3 + 1) = array([0, 1, 2, 3]).

Reply:
> Perfect! Please ping me once you're done :)

@hoechenberger, I am OK with the changes if you and @agramfort are.  Thank you both for your patience.

Reply:
Yes I agree that we need +1 here, since according to the docstring, we're working with a closed interval here